service/src/main/java/eu/europa/ec/fisheries/uvms/mobileterminal/entity/MobileTerminal.java

```diff
@@ -52,9 +52,9 @@ This file is part of the Integrated Fisheries Data Management (IFDM) Suite. The
 	@NamedQuery(name=MobileTerminal.FIND_BY_SERIAL_NO, query="SELECT m FROM MobileTerminal m WHERE m.serialNo = :serialNo"),
 	@NamedQuery(name=MobileTerminal.FIND_BY_ASSET_ID, query="SELECT m FROM MobileTerminal m WHERE m.asset.id = :assetId"),
 	@NamedQuery(name=MobileTerminal.FIND_BY_UNASSIGNED, query="SELECT m FROM MobileTerminal m WHERE m.asset IS NULL"),
-	@NamedQuery(name=MobileTerminal.FIND_BY_DNID_AND_MEMBER_NR_AND_TYPE,
+	@NamedQuery(name=MobileTerminal.FIND_BY_DNID_AND_MEMBER_NR_OR_SERIALNUMBER_AND_TYPE,
             query="SELECT DISTINCT m FROM MobileTerminal m LEFT OUTER JOIN Channel c ON m.id = c.mobileTerminal.id " +
-                    "WHERE m.archived = false AND c.archived = false AND c.dnid = :dnid AND c.memberNumber = :memberNumber AND m.mobileTerminalType = :mobileTerminalType")
+                    "WHERE m.archived = false AND (c.archived = false AND c.dnid = :dnid AND c.memberNumber = :memberNumber OR m.serialNo = :serialNumber) AND m.mobileTerminalType = :mobileTerminalType")
```

Comment:
skriva om den här som:
(c.archived = false AND ((c.dnid = :dnid AND c.memberNumber = :memberNumber) OR m.serialNo = :serialNumber) AND m.mobileTerminalType = :mobileTerminalType")
?